src/test/modelmanager_test.cpp

```diff
@@ -1370,6 +1388,10 @@ TEST_F(ModelManager, HandlingInvalidLastVersion) {
     modelInstanceUnloadGuard.reset();
     ASSERT_EQ(status, ovms::StatusCode::MODEL_VERSION_NOT_LOADED_YET);
 
+#ifdef _WIN32
+    // Manual unload required because OVCORE keeps the file handle opened after modelLoad, preventing the test to remove the directory.
```

Comment:
add FIXME since this is a bug or skip the test so we don;t get wrong impression that everything is fine